src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -79,6 +78,9 @@
     <PackageReference Include="System.Reactive.Linq" Version="5.0.0" />
     <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\WebJobs.Script.Abstractions\WebJobs.Script.Abstractions.csproj" />
```

Comment:
@fabiocav should I revert this and release a new Abstractions package before this PR, then update the version for `Microsoft.Azure.WebJobs.Script.Abstractions`?